lib/services/ios-project-service.ts

```diff
@@ -197,23 +197,33 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 	public isPlatformPrepared(projectRoot: string): IFuture<boolean> {
 		return this.$fs.exists(path.join(projectRoot, this.$projectData.projectName, constants.APP_FOLDER_NAME));
 	}
-
+	
 	public addLibrary(libraryPath: string): IFuture<void> {
 		return (() => {
-			this.validateFramework(libraryPath).wait();
-
+			let extension = path.extname(libraryPath);
+			if (extension === ".framework") {
+				this.addDynamicFramework(libraryPath);
+			} else {
+				this.$errors.failWithoutHelp("The bundle at %s does not appear to be a dynamic framework package.", libraryPath);
```

Comment:
Consider explaining the user how to fix the issue - i.e. - can they use another command?